src/components/Preview/Preview.tsx

```diff
@@ -0,0 +1,7 @@
+import { IPreview } from './Preview.types';
+
+function Preview(props: IPreview) {
+  return <div>{props.previewText}</div>;
```

Comment:
E aí ao invés de usar props, usamos children. **acredito que talvez tenha que mudar a typagem do children que deixa de ser um string para ser um React Node
```suggestion
function Preview(children: IPreview) {
  return <div>{children}</div>;
```

Reply:
Assim acho que da pau man, c pa tem q ser assim
```suggestion
function Preview(props: IPreview) {
  return <div>{props.children}</div>;
```